.travis.yml

```diff
@@ -10,6 +10,7 @@ python:
 install:
 - pip install -r requirements.txt
 - node scripts/fix_abroad_shrinkwrap.js
+- npm config set registry https://registry.npm.taobao.org
```

Comment:
You can just remove "node scripts/fix_abroad_shrinkwrap.js"

Reply:
I mean, by removing "node scripts/fix_abroad_shrinkwrap.js", you will get a shrinkwrap of npm.taobao.org, and you don't need to run `npm config set` any more.